stackanetes/k8s_instance.py

```diff
@@ -34,6 +36,8 @@ def __init__(self, service_name, service_dir):
         self.service_type = service_name.split("-")[0]
         self.service_dir = service_dir
         self._load_service_variables()
+        if not check_if_namespace_exist(CONF.stackanetes.namespace):
```

Comment:
You are right, there is also a solution, but I prefer to use that function anyway. If We execute that creating without checking namespace it will make our deployment log unreadable - we will receive too many Warnings.